samples/csharp_webapi/13.core-bot/Microsoft.LuisRecognizer.schema

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
Why do we have .schema files for this sample?

Reply:
that was an accident!!!
i just pulled the sample, restored packages, fixed a build issue in the sending trace and now i see this schema stuff?

Reply:
do you know where it comes from?

Reply:
so my assumption was this was correct - i.e. something in the packages that nuget restore added